frontend/src/scenes/data-warehouse/new/sourceWizardLogic.tsx

```diff
@@ -438,6 +443,12 @@ export const sourceWizardLogic = kea<sourceWizardLogicType>([
                 actions.closeWizard()
             }
         },
+        createTableSuccess: () => {
+            actions.onClear()
+            actions.clearSource()
+            actions.loadSources(null)
+            actions.resetSourceConnectionDetails()
```

Comment:
I would reuse `closeWizard` but we dont want to redirect as the table logic deals with this - we go to a different page from a manual link, so this duplicated code is fine for now